grapher/color/ColorScale.ts

```diff
@@ -229,9 +229,15 @@ export class ColorScale {
         if (!sortedNumericValues.length) return []
         const sampleMean = mean(sortedNumericValues) as number
         const sampleDeviation = deviation(sortedNumericValues) as number
-        return sortedNumericValues.filter(
+        const withoutOutliers = sortedNumericValues.filter(
             (d) => Math.abs(d - sampleMean) <= sampleDeviation * 2
         )
+
+        if (deviation(withoutOutliers) === 0) {
```

Comment:
super nit: `return deviation(withoutOutliers) === 0 ? sortedNumericValues : withoutOutliers`

Reply:
I wanted to have the comment in there, and thought it was clearer with an explicit if statement.

Reply:
a unit test would be a lot better than a comment (i think that was your earlier point) at some point (no rush)